shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -459,6 +459,7 @@ class JSBackend {
         traits += topLevelScope.declareTrait(name, tparams map { _.name }, body, methods)
       case TypeDef(Cls, TypeName(name), tparams, baseType, _, members, _) =>
         classes += topLevelScope.declareClass(name, tparams map { _.name }, baseType, members)
+      case TypeDef(Nms, TypeName(name), _, _, _, members, _) => () // TODO: deal with namespace
```

Comment:
Silent errors are the worst! Better raise a code-gen exception.

Reply:
I've replaced all silent errors with `throw CodeGenError("Namespaces are not supported yet.")` or `throw new Exception("Namespaces are not supported yet.")`. I used general `java.lang.Exception` when the code is not about the code generation.

Reply:
Ok. You can also simply use `assert` and `require`, depending on the case. More straightforward and clear than explicitly throwing an `Exception`.

Reply:
Since `assert`'s and `require`'s return type is `Unit`, is it ok to append `null` or `td.bodyTy` to make the code pass the type checking?